Sources/App/routes.swift

```diff
@@ -28,7 +30,25 @@ func routes(_ app: Application) throws {
         }
     }
     
-    app.routes.get("admin") { request -> View in
+    app.get("conduct") { req -> EventLoopFuture<View> in
+        return req.view.render("Secondary/conduct", ["name": "Leaf"])
+    }
+
+    try app.routes.register(collection: AuthController()) // TODO: Split this out into web/api/admin
+    try app.routes.register(collection: SpeakerController()) // TODO: Split this out into web/api/admin
+    try app.routes.register(collection: EventsController()) // TODO: Split this out into web/api/admin
+    
+    // MARK: - API Routes
+    
+    let apiRoutes = app.grouped("api", "v1")
```

Comment:
Might be good to separate these out into other route files as this will probably grow a lot as we split API/View

Reply:
Good idea. 😃